src/main/java/nextstep/ladder/domain/Ladder.java

```diff
@@ -0,0 +1,48 @@
+package nextstep.ladder.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Ladder {
+
+    public final Players players;
```

Comment:
```suggestion
    private final Players players;
```
인스턴스 변수는 특별한 이유가 없는 한 private을 구현할 것을 추천